snomed/com.b2international.snowowl.snomed.common/src/com/b2international/snowowl/snomed/common/SnomedConstants.java

```diff
@@ -138,6 +138,15 @@ private Concepts() { }
 		public static final String REFSET_MRCM_ATTRIBUTE_RANGE_ROOT = "723592007";
 		public static final String REFSET_MRCM_ATTRIBUTE_RANGE_INTERNATIONAL = "723562003";
 		
+		public static final Set<String> METADATA_REFSETS = ImmutableSet.of(
+				REFSET_DESCRIPTOR_REFSET,
```

Comment:
There are other structural reference sets like axioms, attribute values, association types, and language refsets. Consider them when creating this set. Another option would be to either configure this set via a config.yml configuration parameter or via RF2 import request parameter.